properties/font/leading.json

```diff
@@ -0,0 +1,20 @@
+{
+	"font": {
+		"leading": {
```

Comment:
should we just call it `lineHeight`?

Reply:
Yea, let's track closer to what it's called in CSS